x-pack/plugins/observability/public/embeddable/slo/alerts/slo_configuration.tsx

```diff
@@ -71,6 +80,21 @@ export function SloConfiguration({ initialInput, onCreate, onCancel }: SloConfig
             />
           </EuiFlexItem>
         </EuiFlexGroup>
+        {hasGroupBy && (
+          <>
+            <EuiSpacer />
+            <EuiSwitch
+              label={i18n.translate(
+                'xpack.observability.sloConfiguration.euiSwitch.showAllGroupByLabel',
```

Comment:
@shahzad31 How about we prefix the copy with `xpack.observability.sloAlertsEmbeddable` to be consistent with rest i18n copies? It would become `xpack.observability.sloAlertsEmbeddable.sloConfiguration.euiSwitch.showAllGroupByLabel` that long though..